ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -262,6 +262,17 @@ function AuthProviders(): ReactElement {
                                             </a>
                                         </ExternalLink>
                                     </Flex>
+                                    <Flex direction={{ default: 'column' }}>
```

Comment:
```suggestion
```
If you remove the closing `</Flex>` above this new section and the opening `<Flex>` in the new code, we can contain both links in a single `<Flex>` and make use of the `default: 'column'` prop that is set, and give the links a bit more spacing automatically.